nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -2197,14 +2197,17 @@ private boolean mayBeNullExpr(VisitorState state, ExpressionTree expr) {
       // obviously not null
       return false;
     }
+    switch (expr.getKind()) {
```

Comment:
Can we make this an `if` rather than a `switch`?

Reply:
i made it a switch in anticipation of more expression types returning early, but ok changed to `if`